test/cpp/end2end/BUILD

```diff
@@ -531,6 +531,24 @@ grpc_cc_test(
     ],
 )
 
+grpc_cc_test(
```

Comment:
this test needs to be set as "manual" for bazel, there's no chance of it working on foundry IMHO.

Reply:
Can you please point me to the configuration to set this to "manual"?